Dockerfile.xdmod

```diff
@@ -0,0 +1,51 @@
+FROM moc-xdmod-base
+
+#COPY assets /tmp/assets
+#RUN /tmp/assets/copy-caches.sh
+#COPY bin /root/bin
+
+WORKDIR /root
+
+
+# This installs a barebones 10.0 instance of XDmod
+RUN mkdir -p /root/rpmbuild/RPMS/noarch \
+    && mkdir /tmp/supremm \
+    && yum -y install openssh-server openssh-clients pcp pcp-manager pcp-conf pcp-libs python-pcp perl-PCP-PMDA pcp-system-tools pcp-pmda-gpfs pcp-pmda-lustre pcp-pmda-infiniband pcp-pmda-mic pcp-pmda-nvidia-gpu \
+    pcp-pmda-nfsclient pcp-pmda-perfevent pcp-pmda-json python3 \
+    && pip3 install --upgrade pip \
+    && pip3 install keystoneauth python-keystoneclient python-novaclient python-cinderclient cryptography python-openstacksdk mysql-connector-python
```

Comment:
The bulk of this block (setting up an `rpmbuild` directory, installing an `ssh` server, and installing all the `pcp` related packages) is unnecessary. Probably we don't need the `ssh` client, either. It looks like really we just need to install `python` and then continue with the `pip` commands.

Reply:
I have eliminate pcp.  However, xdmod seems to require the openssl-server and openssl-clients with the way that they are doing things.  I have simplified this a bit.